data/korath/korath weapons.txt

```diff
@@ -1011,6 +1011,35 @@ effect "firelight ring"
 	"velocity scale" 0.1
 
 
+outfit "Microbot Factory"
+	category "Guns"
+	thumbnail "outfit/unknown"
+	"mass" 112
+	"outfit space" -112
+	"weapon capacity" -112
+	"required crew" 20
+	"gun ports" -1
+	"spinal mount" -1
+	weapon
+		"hardpoint sprite" "hardpoint/factory/microbot factory"
+			"frame rate" 12
+		"hit effect" "shunt-strike hit" 
+		"die effect" "shunt-strike hit" 
+		"inaccuracy" 10
+		"velocity" 10
+		"lifetime" 2
+		"reload" 6
+		"firing energy" 5
+		"blast radius" 100
+		"shield damage" 48
+		"corrosion damage" 3
+		"minable damage" 200
+		"firing hull" -100
```

Comment:
50 energy per second for 1000 hull repair out of 112 tonnes is better than every other hull repair outfit we have.

Reply:
I think you forgot to add the keyword that prevents the player from looting it?

Reply:
Okay, it's bigger and unplunderable now.  Also not matched up at all, I figured it's got some empty space and isn't all weapon.

Reply:
The Refabricator is a big microbot generator/control ship. In other words, the entire ship is the hull repair part of a systems core. It should have a crazy repair rate. Using a weapon is a creative way to do it. The original Refabricator just had built-in hull repair.

Reply:
The negative firing hull should really either have a lower magnitude or the firing energy should be much higher, like at least 20, probably 30x what it currently is.

Reply:
This looks to me like a case of the game doesn't support the mechanics needed to do what we want to do (#2004), so we shouldn't try to force it.